ios/.xcode.env

```diff
@@ -0,0 +1 @@
+export NODE_BINARY=$(command -v node)
```

Comment:
Is this used anywhere?

Reply:
I am not sure, this was automatically generated, but I tend to not think so. Will check it.

Reply:
[react-native-xcode.sh](https://github.com/facebook/react-native/blob/55e4543eea1a2f7a42ed6997c9a692b4fc61550a/packages/react-native/scripts/react-native-xcode.sh#L142) does use it, 
--keep in mind that I had problems with using multiple node env (nvm)-- maybe about my environment